.phan/stubs/wordpress-constants.php

```diff
@@ -80,3 +104,6 @@
 
 // Templating-related WordPress constants.
 define( 'WP_DEFAULT_THEME', 'twentytwentywhenever' );
+
+// Testing constants.
+define( 'WP_TESTS_DOMAIN', 'example.org' );
```

Comment:
Might be worth mentioning that this isn't a constant in WP, but is used in PHPUnit tests.